src/utils/legacyLayerParamUtils.js

```diff
@@ -171,3 +171,22 @@ export async function getKmlLayerFromLegacyAdminIdParam(adminId) {
         kmlMetaData
     )
 }
+
+export function handleBodLayerIdParam(params, store, newQuery) {
+    params.forEach((param_value, param_key) => {
+        let layer = null
+        layer = store.state.layers.config.find((layer) => layer.getID() === param_key)
+        if (layer) {
+            const featuresIds = param_value.split(',').join(':')
+            if (newQuery.layers?.includes(param_key)) {
+                //if the layer already exist, we insert the features id parameter
+                const [layerIdWithCustomParams, visible, opacity] = newQuery.layers.split(',')
+                newQuery.layers = `${layerIdWithCustomParams}@features=${featuresIds},${visible},${opacity}`
```

Comment:
So if I understand correctly, this will lead to something like
`https://.../#/map?layers=my.cool.layer@features=1@features=2,1,1` ?

If so, I think it doesn't make sense to have a plural word for the feature ID, a `@feature=...` would be better IMHO

Reply:
I reworked this whole bit : 
I was doing something wrong, the goal now will be to transform this : 
`.../?layers=layer-id-1,layer-id-2&layer-id-1=1,3,5` into `/#map?layers=layer-id-1@features=1:3:5;layer-id-2`

it will also work with something along the line of this `.../?layer-id-1=1,3,5/#map/layers=layer-id-1@features=1:2:4` to turn it into `/#map?layers=layer-id-1@features=1:2:3:4:5`